app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -38,11 +47,65 @@ def get_jwt_identity
         @authentication_parameters.jwt_identity =  @jwt_configuration.jwt_identity(@authentication_parameters)
       end
 
+      def validate_origin
+        @validate_origin.(
+          account: @authentication_parameters.account,
+          username: @authentication_parameters.username,
+          client_ip: @authentication_parameters.client_ip
```

Comment:
why dont you use delegators?

Reply:
I copied it from authenticate.rb
https://github.com/cyberark/conjur/blob/08428cdc8ab192a69f7c01962655d06ab2b62742/app/domain/authentication/authenticate.rb#L67